template/docs/README.md

```diff
@@ -0,0 +1,160 @@
+website. You can read more about the files and folders created by
+`main.py` on the
+[Outputs](https://sprout.seedcase-project.org/docs/design/interface/outputs)
+page of the design documentation.
+
+### Creating package properties
+
+1. Run `main.py` to create the `scripts/package_properties.py` file for
+    the properties of your Data Package.
+
+    ``` {.bash filename="Terminal"}
+    just build
+    ```
+
+    You can also run `main.py` by clicking the "Run" button in your IDE.
+
+2. Open `scripts/package_properties.py` and fill in all required
+    fields. Also fill in any optional fields you find useful. You can
+    always update these later. Make sure to save the file.
+
+3. In `main.py`, uncomment the lines referencing the
+    `package_properties` and `package_path` variables: lines 4, 17, 29,
+    34, and 36.
+
+4. Rerun `main.py` to create the `datapackage.json` and `README.md`
+    files for your Data Package.
+
+### Creating a new resource
+
+#### With data to add to the resource
+
+If you already have some data in a tidy format, load this as a Polars
+data frame into the `raw_data` variable in `main.py`.
+
+1. Uncomment lines up to and including the creation of resource
+    properties: lines 1, 18, 20, and 23-26.
+
+2. Fill in the `resource_name` argument in line 24.
+
+3. Rerun `main.py` to create the
+    `scripts/resource_properties_<name>.py` file for the properties of
+    the new resource.
+
+4. Open `scripts/resource_properties_<name>.py` and fill in all
+    required fields. Also fill in any optional fields you find useful.
+    You can always update these later. Make sure to save the file.
+
+5. In `package_properties.py`, import your new resource properties by
+    uncommenting line 3 and updating it with the name of your resource.
+    Also uncomment the `resources` field in lines 46-48 and update the
+    name of the resource properties in the array to match the name of
+    your new resource.
+
+6. In `main.py`, import your new resource properties by uncommenting
+    line 6 and updating it with the name of your resource.
+
+7. Uncomment all remaining lines in the file and rename the
+    `resource_properties` variable to the name of the new resource
+    properties you just imported.
+
+8. Rerun `main.py`. This will:
+
+    - Update `datapackage.json` and `README.md`.
+    - Create a `resources/` folder containing a folder for your new
+        resource. In here, you will find a `batch/` folder with the
+        individual data batches you've uploaded for this resource and a
+        `data.parquet` file containing all resource data.
+
+#### Without data to add to the resource
+
+You can create a new resource without adding data to it using a shorter
+version of the steps above.
+
+- Step 1: uncomment only lines 23, 24, and 26.
+- Step 4: if you cannot describe columns in your data at this stage,
+    comment out the `schema` field.
+- Steps 6-7: as you have no data to add, you should skip these steps.
+
```

Comment:
```suggestion
```

Let's not get into describing situations where they don't have data, it gets complicated and we don't have a good flow there yet.